### PR TITLE
Beekeeping fixes and tweaks

### PR DESCRIPTION
**Pull request by NanakoAC**
This was meant to be a 5 minute project but it took two days as i kept discovering more issues.

In short, baymerge changed beekeeping. Generally for the better, i love some of the new things they did. But they also removed some things (like the actual bee mobs) and some of their new stuff wasn't properly implemented. 
So this PR fixes up both the bay beekeeping content and ours, to work in harmony together, remove redundant stuff, and fix lots of bugs.

I made the beekeeping crate have all the necessary new equipment, notably including the honey extractor which isn't mapped in. .Gave it a big wooden crate so it doesn't seem dumb to have machinery inside it. Also added a general (civilian, non departmental) biosuit to it. it looks almost exactly like a beekeeping suit

I replaced bay's dumb bee smoker (which was essentially just a context sensitive quest item) with an improved bee smoker which is a decently interesting tool. It spews directed clouds of smoke to calm bees, actually requires a little skill to use, its very fun.  and has potential to use for creating a distraction to escape firefights too. It runs on welding fuel to create the smoke. added them to the cargo spawning pool

Fixed up the myriad of bugs with the bee net and re-enabled it, put a couple in the bee crate. It can now successfully scoop up bees, release them, or, most importantly, deposit them back into a hive. Requires coodinated use with the smoker to perform optimally.

Re-enabled the bee mobs spawning from beehives. For some reason bay decided to remove them, and instead replace any instance where bees might spawn, with a polite error message dening you from doing the thing. Ive made angry bees spawn when honey is harvested without smoking them first, when the hive is moved without smoking, and when its deconstructed. I also made docile, non-angry bees come out randomly if you leave the lid open, herding them adds some fun gameplay.

Rewrote the beekeeping manual and tossed it in.  A lot of these mechanics are somewhat unintuitive and it should help
Slightly reduced bee damage.
Majorly reduced volume and frequency of their sounds Bee sounds are awesome but its not fun for them to be deafening from half a station away

All in all, makes beekeeping a more well rounded and complete mechanic. i'm pretty happy with this

### CODE REVIEW

File: code/controllers/Processes/mob.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 /datum/controller/process/mob/doWork()
 	for(last_object in mob_list)
 		var/mob/M = last_object
-		if(isnull(M.gcDestroyed))
+		if(M && isnull(M.gcDestroyed))
```
Comment (NanakoAC) [inline code/controllers/Processes/mob.dm]: this change to the mob controller was added after discussion with skull. Bees managed to turn up an unusual edge case bug and this was the best means of preventing it

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -1371,7 +1377,7 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 
 /datum/supply_packs/hydrotray
 	name = "Empty hydroponics tray"
-	cost = 30
+	cost = 10
```
Comment (NanakoAC) [inline code/datums/supplypacks.dm]: the cost on this was just stupidly high, call it a minor fix.

You need like eight of these things to setup a garden, theyre not very good individually

File: code/modules/mob/living/simple_animal/bees.dm [hunk 3]
```diff
@@ -14,14 +16,16 @@
...
+	var/loner = 0
 	pass_flags = PASSTABLE
 	turns_per_move = 6
 	var/obj/machinery/portable_atmospherics/hydroponics/my_hydrotray
 
-/mob/living/simple_animal/bee/New(loc, var/obj/machinery/apiary/new_parent)
+/mob/living/simple_animal/bee/New(loc, var/obj/machinery/beehive/new_parent)
 	..()
 	parent = new_parent
+	bee_list["\ref[src]"] = src
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/bees.dm]: Debugging leftover?

File: code/modules/mob/living/simple_animal/bees.dm [hunk 4]
```diff
@@ -1,4 +1,6 @@
 //Bees are spawned from an apiary, and will slowly die if it is destroyed.
+var/global/list/bee_list = list()
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/bees.dm]: Debugging leftover?

**Pull request merged**